lib/measured/rails/validations.rb

```diff
@@ -6,13 +14,40 @@ def validate_each(record, attribute, measurable)
     measurable_unit = record.send("#{ attribute }_unit")
     measurable_value = record.send("#{ attribute }_value")
 
-    message = options[:message] || "is not a valid unit"
+    return unless measurable_unit.present? || measurable_value.present?
```

Comment:
Should this be a straight up return? I feel like it might make sense to make measurable_unit and measurable_value required.